test/Elastica/Base.php

```diff
@@ -135,7 +135,7 @@ protected function _checkScriptInlineSetting()
         $nodes = $this->_getClient()->getCluster()->getNodes();
         $scriptInline = $nodes[0]->getInfo()->get('settings', 'script', 'inline');
 
-        if ($scriptInline != 'true' && $this->_getVersion() < 6) {
+        if ($this->_getVersion() < 6 && $scriptInline != 'true') {
```

Comment:
There is no logical change here, right?

Reply:
no to be honest I'll be evaluate in a next PR if we still need _checkScriptInlineSetting()
as ES6 has script inline and stored allowed by default. So no need to enable them by elasticsearch.yml